trio/_core/_run.py

```diff
@@ -1233,7 +1236,7 @@ def run(
       args: Positional arguments to be passed to *async_fn*. If you need to
           pass keyword arguments, then use :func:`functools.partial`.
 
-      clock: ``None`` to use the default system-specific monotonic clock;
```

Comment:
This is just referring to the general concept of a "monotonic clock", so should stay the same.